tests/testthat/test-utils.R

```diff
@@ -1,5 +1,63 @@
 context('utils')
 
+test_that("numBinsToBinWidth() returns a binWidth that will actually provide the desired bins.", {
+  binWidth <- numBinsToBinWidth(testDF$entity.int6, 8)
+  viewport <- findViewport(testDF$entity.int6, 'NUMBER')
+  x <- bin(testDF$entity.int6, binWidth, viewport)
+  expect_equal(data.table::uniqueN(x),6)
+
+  binWidth <- numBinsToBinWidth(testDF$entity.int6, 1)
+  viewport <- findViewport(testDF$entity.int6, 'NUMBER')
+  x <- bin(testDF$entity.int6, binWidth, viewport)
+  expect_equal(data.table::uniqueN(x),1)
+  
+  binWidth <- numBinsToBinWidth(testDF$entity.contA, 8)
+  viewport <- findViewport(testDF$entity.contA, 'NUMBER')
+  x <- bin(testDF$entity.contA, binWidth, viewport)
+  expect_equal(data.table::uniqueN(x),8)
+
+  binWidth <- numBinsToBinWidth(testDF$entity.contA, 27)
+  viewport <- findViewport(testDF$entity.contA, 'NUMBER')
+  x <- bin(testDF$entity.contA, binWidth, viewport)
+  #expect_equal(data.table::uniqueN(x),27)
```

Comment:
fiddled around a bit to try to find a pattern. Only learned that swapping contA for contB is even weirder